docs/src/tutorial.rst

```diff
@@ -176,15 +176,20 @@ values:
 
         // list of properties in this dataset
         "properties": {
-            // each property have a name, a target and some values
+            // each property have at least a name, a target and some values
```

Comment:
I would be more thorough here. For example, each property must have at least a name, a target, and some values. Optional entries for the units and descriptions can be added to the property.

Reply:
Sounds good! I've added this